desktop_version/src/Render.cpp

```diff
@@ -2158,6 +2158,90 @@ static const char* interact_prompt(
     return buffer;
 }
 
+static void mode_indicator_text(const int alpha)
+{
+    const int flags = PR_BRIGHTNESS(alpha) | PR_BOR;
+    const int r = 220 - help.glow;
+    const int g = 220 - help.glow;
+    const int b = 255 - help.glow/2;
+    const int x = 5;
+    const int spacing = font::height(flags) + 2;
+    int y = 5;
+    if (game.advancetext)
+    {
+        /* Prevent clashing */
+        y += 15;
+    }
+
+    /* FIXME: Some strings have not yet been translated. In order to not have
+     * English text in other languages, they are substituted with existing
+     * ones. Remove all substitute text when they're fully translated. */
+
+    if (map.invincibility)
+    {
+        const char* english = "Invincibility mode enabled";
+        const char* text = loc::gettext(english);
+        if (loc::lang != "en" && SDL_strcmp(english, text) == 0)
+        {
+            /* Substitute text */
+            text = loc::gettext("Invincibility is ON.");
+        }
+        font::print(flags, x, y, text, r, g, b);
+        y += spacing;
+    }
+
+    enum GlitchrunnerMode mode = GlitchrunnerMode_get();
+    if (mode != GlitchrunnerNone)
+    {
+        char buffer[SCREEN_WIDTH_CHARS + 1];
+        const char* mode_string = loc::gettext(GlitchrunnerMode_enum_to_string(mode));
+        vformat_buf(
+            buffer, sizeof(buffer),
+            loc::gettext("Glitchrunner mode is {version}"),
+            "version:str", mode_string
+        );
+        font::print(flags, x, y, buffer, r, g, b);
+        y += spacing;
+    }
+
+    if (graphics.flipmode)
+    {
+        const char* english = "Flip Mode enabled";
+        const char* text = loc::gettext(english);
+        char buffer[SCREEN_WIDTH_CHARS + 1];
+        if (loc::lang != "en" && SDL_strcmp(english, text) == 0)
+        {
+            /* Substitute text */
+            SDL_snprintf(
+                buffer, sizeof(buffer), "%s %s",
+                loc::gettext("Flip Mode"),
+                loc::gettext("Currently ENABLED!")
+            );
```

Comment:
Concatenating translations together in new contexts is a bad idea.

In this case, speaking for Dutch and other languages, starting the word "Currently" with a capital letter, in the middle of the sentence, is painful.

I wouldn't be surprised if you get weird wordings like "Flip mode This is currently ENABLED!" in some languages, and I also wouldn't be surprised if it runs offscreen sometimes. Just "Flip Mode" should be fine - by itself it shows that it's enabled.